surfman/build.rs

```diff
@@ -15,8 +15,9 @@ fn main() {
         windows: { target_os = "windows" },
         macos: { target_os = "macos" },
         android: { target_os = "android" },
+        ohos: { target_env = "ohos" },
         // TODO: is `target_os = "linux"` the same as the following check?
-        linux: { all(unix, not(any(macos, android))) },
+        linux: { all(unix, not(any(macos, android, ohos))) },
```

Comment:
I believe it might be a good opportunity to rename the `linux` cfg option to something that makes it more clear, that it is meant for unix systems with x11 or wayland. But I'm not sure what a good name would be.

Reply:
I think this script is just reusing what winit has. Not many other UNIX-like OSes care about desktop environments usually. It's okay to stay linux IMHO.

Reply:
If it is indeed based on winit, then winit has since changed the names - there is now e.g. `free_unix`, `x11` and `wayland`. 
https://github.com/rust-windowing/winit/blob/master/build.rs
I think it would make sense to rename our cfg flags here to match the ones from `winit` - would you agree?

Reply:
I opened a seperate PR for the `cfg` changes in https://github.com/servo/surfman/pull/290